### ISSUE TITLE
export of BTE results to graphml format

### ISSUE THREAD

**Issue by andrewsu** [biothings/biothings_explorer#38]

**Comment by kevinxin90** [comment]
fixed in 605639a384428fa2891c8e66192f3c86e889067a

**Comment by kevinxin90** [comment]
**Example python code**

```{r test-python, engine='python'}
from biothings_explorer.hint import Hint
from biothings_explorer.user_query_dispatcher import FindConnection
cxcr4 = ht.query('CXCR4')['Gene'][0]
fc = FindConnection(input_obj=cxcr4, output_obj='Protein', intermediate_nodes=None)
fc.connect(verbose=True)
fc.to_graphml()
```

**GraphML output**
```
<?xml version='1.0' encoding='utf-8'?>
<graphml xmlns="http://graphml.graphdrawing.org/xmlns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://graphml.graphdrawing.org/xmlns/1.0/graphml.xsd"><key id="d20" for="edge" attr.name="$api" attr.type="string"/>
<key id="d19" for="edge" attr.name="$input" attr.type="string"/>
<key id="d18" for="edge" attr.name="bts:uniprot" attr.type="string"/>
<key id="d17" for="edge" attr.name="bts:ensembl" attr.type="string"/>
<key id="d16" for="edge" attr.name="@type" attr.type="string"/>
<key id="d15" for="edge" attr.name="source" attr.type="string"/>
<key id="d14" for="edge" attr.name="label" attr.type="string"/>
<key id="d13" for="node" attr.name="bts:uniprot" attr.type="string"/>
<key id="d12" for="node" attr.name="bts:symbol" attr.type="string"/>
<key id="d11" for="node" attr.name="bts:name" attr.type="string"/>
<key id="d10" for="node" attr.name="bts:pharmgkb" attr.type="string"/>
<key id="d9" for="node" attr.name="bts:unigene" attr.type="string"/>
<key id="d8" for="node" attr.name="bts:umls" attr.type="string"/>
<key id="d7" for="node" attr.name="bts:pharos" attr.type="string"/>
<key id="d6" for="node" attr.name="bts:entrez" attr.type="string"/>
<key id="d5" for="node" attr.name="bts:omim" attr.type="string"/>
<key id="d4" for="node" attr.name="bts:hgnc" attr.type="string"/>
<key id="d3" for="node" attr.name="bts:ensembl" attr.type="string"/>
<key id="d2" for="node" attr.name="level" attr.type="long"/>
<key id="d1" for="node" attr.name="identifier" attr.type="string"/>
<key id="d0" for="node" attr.name="type" attr.type="string"/>
<graph edgedefault="directed"><node id="CXCR4">
  <data key="d0">Gene</data>
  <data key="d1">bts:entrez</data>
  <data key="d2">1</data>
  <data key="d3">ENSG00000121966</data>
  <data key="d4">2561</data>
  <data key="d5">162643</data>
  <data key="d6">7852</data>
  <data key="d7"></data>
  <data key="d8">C1332823</data>
  <data key="d9">Hs.593413</data>
  <data key="d10">PA27058</data>
  <data key="d11">C-X-C chemokine receptor type 4,LPS-associated protein 3,leukocyte-derived seven transmembrane domain receptor,neuropeptide Y receptor Y3,seven transmembrane helix receptor,seven-transmembrane-segment receptor, spleen,chemokine receptor,C-X-C motif chemokine receptor 4,CD184 antigen,neuropeptide Y3 receptor,fusin,lipopolysaccharide-associated protein 3,stromal cell-derived factor 1 receptor,chemokine (C-X-C motif) receptor 4,SDF-1 receptor</data>
  <data key="d12">CXCR4</data>
</node>
<node id="ENSP00000241393">
  <data key="d1">bts:ensembl</data>
  <data key="d0">Protein</data>
  <data key="d2">2</data>
  <data key="d3">ENSP00000241393</data>
</node>
<node id="ENSP00000386884">
  <data key="d1">bts:ensembl</data>
  <data key="d0">Protein</data>
  <data key="d2">2</data>
  <data key="d3">ENSP00000386884</data>
</node>
<node id="P61073">
  <data key="d1">bts:uniprot</data>
  <data key="d0">Protein</data>
  <data key="d2">2</data>
  <data key="d13">P61073</data>
</node>
<edge source="CXCR4" target="ENSP00000241393" id="0">
  <data key="d14">bts:hasGeneProduct</data>
  <data key="d15">null</data>
  <data key="d16">Protein</data>
  <data key="d17">ENSP00000241393,ENSP00000386884</data>
  <data key="d18">P61073</data>
  <data key="d19">bts:ensembl</data>
  <data key="d20">mygene.info</data>
</edge>
<edge source="CXCR4" target="ENSP00000386884" id="0">
  <data key="d14">bts:hasGeneProduct</data>
  <data key="d15">null</data>
  <data key="d16">Protein</data>
  <data key="d17">ENSP00000241393,ENSP00000386884</data>
  <data key="d18">P61073</data>
  <data key="d19">bts:ensembl</data>
  <data key="d20">mygene.info</data>
</edge>
<edge source="CXCR4" target="P61073" id="0">
  <data key="d14">bts:hasGeneProduct</data>
  <data key="d15">null</data>
  <data key="d16">Protein</data>
  <data key="d17">ENSP00000241393,ENSP00000386884</data>
  <data key="d18">P61073</data>
  <data key="d19">bts:ensembl</data>
  <data key="d20">mygene.info</data>
</edge>
</graph></graphml>
```
